tests/e2e/index.spec.ts

```diff
@@ -61,9 +64,20 @@ test.describe("Index page", () => {
     });
 
     test("tooltips are shown", async ({ page }) => {
-        const firstRegion = await getFirstRegion(page);
+        const firstRegion = await getNthRegion(page, 1);
         await firstRegion.hover();
         await expect(await page.innerText(".leaflet-tooltip-pane")).toContain("Bengo");
         await expect(await page.innerText(".leaflet-tooltip-pane")).toContain("FOI: 0.0134");
     });
+
+    test("selecting country fades colours of other countries", async ({ page }) => {
+        const firstRegion = await getNthRegion(page, 1);
+        const colour = await firstRegion.getAttribute("fill");
+        const stroke = await firstRegion.getAttribute("stroke");
+        const secondRegion = await getNthRegion(page, 1000);
+        await secondRegion.click();
+        await page.waitForURL(/dengue\/may24\/FOI/i);
```

Comment:
Do we need this? It would match immediately?

Reply:
yes sorry this was wrong, fixed now